addon/mixins/dom.js

```diff
@@ -141,14 +141,25 @@ export default Mixin.create({
 
   willDestroyElement() {
```

Comment:
remove willDestroyElement, if the component called didInsertElement, willDestroy will be called after willDestroyElement, and the remove closes over the element.